k8s/trillian/Makefile

```diff
@@ -0,0 +1,93 @@
+include ../app.Makefile
+include ../crd.Makefile
+include ../gcloud.Makefile
+include ../var.Makefile
+
+TAG ?= latest
```

Comment:
Can we use 1.2.1, as in the Application resource definition?

Reply:
Done.